index.html

```diff
@@ -717,27 +717,30 @@
   </div>
   <!-- ends Join files together -->
 
-  <!-- Play image sequance -->
+  <!-- Play image sequence -->
   <span data-toggle="modal" data-target=".play_im_sq"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Play an image sequence directly as moving images">Play an image sequence</button></span>
   <div class="modal fade play_im_sq" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
     <div class="modal-dialog modal-lg">
       <div class="modal-content">
         <div class="well">
           <h3>Play an image sequence</h3>
           <p>Play an image sequence directly as moving images, without having to create a video first.</p>
-          <p><code>ffplay <i>input_file_%06d.ext</i></code></p>
+          <p><code>ffplay -framerate 5 <i>input_file_%06d.ext</i></code></p>
```

Comment:
Hmm… I don’t know. 5 is like a very fast diapositive show, therefore my question. Yet I’m fine with 5 too!